replication-entity/replication-entity-controller/src/main/resources/templates/replication/admin/page/controller.vm

```diff
@@ -99,6 +100,22 @@
   </select>
 #end
 
+#macro(selectDirection $currentDirection, $inputId, $showDirectionSelect)
+  #set ($cssClass = '')
+  #if (!$showDirectionSelect)
+    #set ($cssClass = 'hidden')
+  #end
+  <label class="direction-label $escapetool.xml($cssClass)">$escapetool.xml($services.localization.render('replication.entity.direction'))</label>
+  <select id="$inputId" name="$inputId" class="direction-select $escapetool.xml($cssClass)">
+    #foreach($direction in ['BOTH', 'SEND_ONLY', 'RECEIVE_ONLY'])
+      <option value="$escapetool.xml($direction)"
+        #if($currentDirection == $direction)selected#end>
+        $escapetool.xml($services.localization.render("replication.entity.direction.${direction}"))
```

Comment:
It would probably worth having the desciption of the level as `title` too.

Reply:
I will add the description